### ISSUE TITLE
embedded8266; No Output to WS2812

### ISSUE THREAD

**Issue by Louseman** [cnlohr/colorchord#4]
Trying to get the colorchord/embedded8266 code up and running, but I am not getting any output to
the  ws2812 strip.  I can see data in the oscilloscope, but never anything on the DFT canvas, and no
output to the strip.
I did previously get the esp8266ws2812i2s code running, and the strip works with the "top" program as
expected.
Any hints? -- Thanks.

**Comment by cnlohr** [comment]
Yes, in fact! Several hints.  Check the "parameters" section, click the thing to make it go back to factory settings then save.  I believe the system currently defaults to having the DFT off.

Also, make sure output is not disabled.

**Comment by Louseman** [comment]
Ah, that did it.  This is spectacular.  Thanks!!!

**Comment by lightsarefun** [comment]
Hi, I see data in the oscilloscope and I do see response on DFT and LEDs webpage section, but my LED strip is all yellow (even when no sound is playing). I tried esp8266ws2812i2s and can control the strip just fine. Any thoughts?

Also working though the reset loop when in station mode. Works fine when connected to AP, but when it connects to my network i can see response through webpage in about 10 second intervals before restart. I've read through all the issues and tried various suggestions.

**Comment by cnlohr** [comment]
I don't understand... are you having two separate issues?  Also... do you have a scope or anything you can look at the RX line with? I am curious if something is totally hosed..

Also try ```make burnitall```

**Comment by lightsarefun** [comment]
Hi, Yes two separate issues. 
I really need to get a scope, but after reading "Clearing rest of buffer in ws2812_push #70", I now have response on LED strip by using DWS2812_THREE_SAMPLE instead of DWS2812_FOUR_SAMPLE.
when trying to do `make all` when i was getting the following error:
> user/ws2812_i2s.c:531:42: error: 'WS_BLOCKSIZE' undeclared (first use in this function)
>   while( bufferpl < &((uint8_t*)i2sBlock)[WS_BLOCKSIZE] ) *(bufferpl++) = 0;
Fixed this by adding `#define WS_BLOCKSIZE 4000` to ws2812_i2s.h


The second issue is the esp8266 stuck in a reset loop when i connect to my network. There are other issue threads that have more relevant info about this so i will follow up there.

**Comment by cnlohr** [comment]
There are two known issues that can contribute to the reboot loop, but both that I'm aware of are solved by ```burnitall``` which reflashes even the configuration section of the ESP.

**Comment by lightsarefun** [comment]
i have been using `burnitall`, but still having the issue. I recall seeing a thread about someone having issues when using station mode by connecting via "scan for wifi", rather than manually configuring. I recall trying to manually enter network configs in addition to using the scan feature, but no luck. I tried it on multiple nodeMCU boards, but it is still hosed. Glad to get the LEDs working though, awesome project! Reminds me a much better version of a chromatic tuner project I did in school.
Won't be available to work on this until next week, but I'll post more info about the reset loop.
